app/controllers/api/v1/cookbooks_controller.rb

```diff
@@ -28,6 +28,10 @@ def index
     if params[:user]
       @cookbooks = @cookbooks.owned_by(params[:user])
     end
+
+    if params[:platforms].present? && !params[:platforms][0].blank?
```

Comment:
Was there a reason you opted for `.present?` and then `! .blank?` here? `.present?` in both tests reads better to me.

Also, the doc comment for this method could be updated to include an example of using this new parameter.